chall11/cdorta-n.c

```diff
@@ -0,0 +1,197 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   cdorta-n.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: cdorta-n <cdorta-n@student.42madrid.com    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/05/21 13:42:55 by cdorta-n          #+#    #+#             */
+/*   Updated: 2020/05/21 21:44:23 by cdorta-n         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+
+/*
+** 'Label' -> Positive Integer
+** 'goto' -> goto-statement
+** 'whitespaces' -> only 1 space
+** 'end-of-line' -> \n
+*/
+
+char	*ft_goto_error()
+{
+	char *error = "Infinite loop !";
+	return (error);
+}
+
+int		ft_goto_statement(const char *code, int num, int c)
```

Comment:
You don't have to recode all this functions ! You can just use the official `atoi()` or similar functions in the challenges :) it's even advised, as you would have to use them in a real world environment.